### PR TITLE
Add setup instructions for using GitLab instead of GitHub

### PR DESCRIPTION
**Pull request by BenRamchandani**
For the M1 project exercise for the UBS cohort.

### CODE REVIEW

File: Tools/gitlab.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+![GitLab create a blank project screen](../assets/images/gitlab_new_project.png)
+
+Follow the "Command line instructions" on the GitLab page for your new project to push your existing repository, the commands should look something like:
+
+```sh
+git remote remove origin # Unlink the original GitHub repository
+git remote add origin https://gitlab.com/your-username/devops-project-exercise.git # Replace "your-username" as appropriate, and make sure the repository name is correct.
+git push --set-upstream origin master # Push the current branch
+```
```
Comment (BenRamchandani) [inline Tools/gitlab.md:36]: The new project splash page instructions are a bit more complicated than GitHub's, and don't really look much like this, but it should be enough for them to work it out.

Comment (hugh-emerson) [reply Tools/gitlab.md:36]: Then should you say to ignore GitLab's instructions?

**Comment by BenRamchandani** [comment]
Notes:
* Similar to GitHub you're required to use a Personal Access Token or SSH if you sign in with SSO or enable 2FA. I haven't mentioned this, but we'll need to look out for it.
* I directly encouraged them to use a public repository, so haven't provided a list of our GitLab usernames, partly because I don't have them. I've added a column to the trainers page in Notion for them: https://www.notion.so/ryielle/de03a7705c6845f794feaadfc3050cf0

File: Tools/gitlab.md (hunk 1 above)
Comment (BenRamchandani) [reply Tools/gitlab.md:36]: Yeah, probably, this is what the page looks like 
image

**Pull request merged**